financialaid/templates/review_financial_aid.html

```diff
@@ -1,116 +1,276 @@
+                     style="height: 35px; border-radius: 18px; border: 1px solid rgba(180, 180, 180, 0.5)">
               </td>
-              <td>
-                <b>{{ financial_aid.user.profile.first_name }} {{ financial_aid.user.profile.last_name }}</b>
+              <!-- /Profile picture -->
+              <!-- Name/location -->
+              <td style="white-space: nowrap">
+                <span id="full-name-{{ financial_aid.id }}">
+                  <b>{{ financial_aid.user.profile.first_name }} {{ financial_aid.user.profile.last_name }}</b>
+                </span>
                 <br>
                 <small>{{ financial_aid.user.profile.city }}, {{ financial_aid.user.profile.country }}</small>
               </td>
+              <!-- /Name/location -->
+              <!-- Income -->
+              <td style="white-space: nowrap">
+                <span class="income-usd"><b>$</b>{{ financial_aid.income_usd|floatformat:0 }}</span>
+                <span class="income-local" style="display: none;">
+                  {{ financial_aid.original_income|floatformat:0 }} <b>{{ financial_aid.original_currency }}</b>
+                </span>
+              </td>
+              <!-- /Income -->
+              <!-- Date calculated -->
+              <td>{{ financial_aid.created_on|date:"SHORT_DATE_FORMAT" }}</td>
+              <!-- /Date calculated -->
+              <!-- Date docs sent -->
+              {% if selected_status == statuses.DOCS_SENT %}
+              <td>{{ financial_aid.date_documents_sent|date:"SHORT_DATE_FORMAT" }}</td>
+              {% endif %}
+              <!-- /Date docs sent -->
+              <!-- Adjusted cost -->
               <td>
-                <select class="form-control">
+                {% if selected_status == statuses.PENDING_MANUAL_APPROVAL %}
+                <select id="tier-program-id-{{ financial_aid.id }}" class="form-control">
                 {% for tier_program in tier_programs %}
                   <option value="{{ tier_program.id }}"
                           {% if tier_program == financial_aid.tier_program %}selected{% endif %}>
                     ${{ tier_program.adjusted_cost }}
                   </option>
                 {% endfor %}
                 </select>
+                {% else %}
+                <input type="text" class="form-control" value="${{ financial_aid.adjusted_cost }}" disabled>
+                {% endif %}
               </td>
-              <td>${{ financial_aid.income_usd }}</td>
-              <td>{{ financial_aid.created_on|date:"SHORT_DATE_FORMAT" }}</td>
+              <!-- /Adjusted cost -->
+              <!-- Justification -->
+              {% if selected_status == statuses.PENDING_MANUAL_APPROVAL %}
               <td>
-                <button class="btn btn-default">
-                  <span class="glyphicon glyphicon-ok" style="color: #0074e1"></span>
+                {% if selected_status == statuses.PENDING_MANUAL_APPROVAL %}
```

Comment:
Ah, this was intentional, but I messed up fixing an earlier bug.  This is to show the justification for approved applications, but render the `<select>` if it's pending manual approval.  I'll make the change.